frontend/src/Live.js

```diff
@@ -116,7 +116,7 @@ function Charts({cameras}) {
         if (!cameras) {
             return
         }
-        const url = `/static/data/${cameras[0]['id']}/objects_log/${new Date().toISOString().slice(0, 10)}.csv`;
+        const url = `${cameras[0]['storage_host']}/static/data/${cameras[0]['id']}/objects_log/${new Date().toISOString().slice(0, 10)}.csv`;
```

Comment:
Thanks @mats-claassen for the changes :) 

I was doing a little bit testing, ran API on 8000 and the GUI on 8003, it fails loading the videos and the plot, I tried debugging it seeing network requests in Firefox, seems the link to the csv file generated here is on port 8003 ( which is GUI's) not on 8000 (which is API's). I have no idea if this url string should refer to 8000 or this request from 8003 should internally be translated to 8000, but anyhow, the csv file is available on 8000 with the same url. 

Do you have any idea how to solve this?

Reply:
That url should point to the API, and not the GUI. Also `${cameras[0]['storage_host']}` should be something like `http://0.0.0.0:8000` in that case. Is that what you are seeing? Are you running everything on localhost?

Reply:
Then it is not the right url, it points to the GUI, and no, I am seeing that it is something like http://MYVPSIP:8003/static/data/default/objects_log/2020-06-25.csv , for example. while the API is running on the 8000. And yes both is running on the local host (my vps)

Reply:
@mats-claassen do you have any solution in mind? 
Let me know if you have tested and got different result

Reply:
So I have just tested it again. I run both frontend and API on the same host but access the web panel from another one. That is a small difference to your set up. Also I have changed two lines in the code (which are not pushed) to make this work: I changed the processor host to the correct IP in `config-frontend.ini` and I set `ENV DEV_ALLOW_ALL_ORIGINS=true` in `x86-openvino.Dockerfile` to allow access from that other host.
I could also try running everything on the same host.

Reply:
So, I tested doing everything from the same host and I still need to allow all origins in the processor node ( the `ENV DEV_ALLOW_ALL_ORIGINS=true` part). There is a note in the code with a warning regarding enabling this so I am hesitant to add it to all Dockerfile (jetson, Coral, Openvino, etc.). But I don't know if there is an alternative.

Reply:
Thanks for your help! Concurrently found several other issues with my setup which was the root of my confusion (frontend was also being copied in the processor dockerfiles - I am separating the API and Engine for the Processor too - I will fix this for all devices so don't worry).

The problem is now I get error 405 when the csv file is requested, and I guess this is the same thing you are telling with the `ENV DEV_ALLOW_ALL_ORIGINS=true`. I am testing and will tell you the result.

Reply:
@mats-claassen I could not get it to work with `ENV DEV_ALLOW_ALL_ORIGINS=true` in the dockerfile, would you please push your working state so I can be sure I am doing the whole process right?

Reply:
I still get 405 Error on retrieving the csv file (but the file is being downloaded if I copy and paste its link into my browser.
 [Screen shot](https://ibb.co/v39d6Df)

Reply:
The weird thing is that you get a 200 for the video playlist but can't access the csv. The only change I have is to add this line:
`ENV DEV_ALLOW_ALL_ORIGINS=true`
at line 82 of the Openvino Dockerfile. I will also test this on the x86 without Openvino and let you know but the change should be very similar

Reply:
Thanks a lot @mats-claassen for your help. It is working for your PR, and as my scenario was a little bit different, I could also get it to work with it ( I was trying to separate API and Engine in the processor, made a [PR here on another fork](https://github.com/emma-w-dev/smart-social-distancing/pull/1). )

@mhejrati  I think you can merge this PR in a temporary branch, so @emma-w-dev can make another PR on top of it, including further development of the new architecture with all the modules separated. Then there will be a huge PR after all to apply it to the master.

Reply:
Thanks @JsonSadler. Regarding the separation of API and Engine: how do you envision the communication between API and Engine for say, modify the configurations of the engine as in https://github.com/neuralet/neuralet/issues/25?
Which is the advantage of separating those two?